lib/internal/test_runner/test.js

```diff
@@ -259,7 +259,7 @@ class Test extends AsyncResource {
       skip = '\'only\' option not set';
     }
 
-    if (skip) {
+    if (skip || todo) {
```

Comment:
Why are you making a TODO into a noop? A TODO test should still run, but not fail the test suite if the test fails.

Reply:
hmm, I think it never has behaved that way :/ anyway I,l revert this part and keep only the reporter fixes

Reply:
Just verified that todo tests show up as todo in the summary, but incorrectly set the process exit code.

Reply:
push a fix in a separate commit